WordPress/openverse
# Update logo nav component in header

## Fixes



Related to #476 by @fcoveram 

## Description


This PR updates the `VHomeLink` component and the `VLogoButton` component to match the Figma mockups. It cleans up the components from unused props and adds the Storybook stories.

This PR is necessary for replacing the svg dependencies in #2135, which, in turn, is required for Nuxt 3 migration. This is the reason for "high" priority label.

Because this PR changes the spacing of the header logo, there are a lot of snapshot changes.

## Testing Instructions

Check the home links in the internal header (the one with the page links, as on the homepage, content pages such as "about" page and the single result pages). The logo should be 18px high, as in the mockups, and the link itself should be 48px high. On hover on white pages, the background becomes yellow. When focused, it should have a pink focus ring.
A special case is a pages modal on mobile width screen. You can open it if you open the site on a narrow screen and click on the "hamburger" menu button. The modal background is black, so the logo becomes white. The focus ring is yellow instead of pink.

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
